src/index.tsx

```diff
@@ -16,7 +16,7 @@ type ExcalidrawTextElement = ExcalidrawElement & {
 const LOCAL_STORAGE_KEY = "excalidraw";
 const LOCAL_STORAGE_KEY_STATE = "excalidraw-state";
 
-var elements = Array.of<ExcalidrawElement>();
+let elements = Array.of<ExcalidrawElement>();
```

Comment:
Since you're changing it, might as well make it const :)

Reply:
`elements` are reassigned in the `restore` function

Reply:
arg, would you mind rewriting the assignment in restore to mutate the array in place?

```js
elements.splice(0, elements.length, ...newElements);
```

This way `const` would work and prevent mistakes where people accidentally capture it and it gets stale.